source/connection-strings/mongodb-database-connection-strings.txt

```diff
@@ -1,3 +1,125 @@
+.. _rm-mongodb-database-connection-strings:
+
 ===================================
 MongoDB Database Connection Strings
 ===================================
+
+.. include:: /includes/uri-usage-disclaimer.rst
+
+To start a migration job, Relational Migrator must connect to your MongoDB 
+database. Relational Migrator requires a username and password which has ``readWrite`` 
+access in the target database. This page describes 
+the procedure to make a authenticated user account and the Uniform Resource 
+Identifier (URI) formats for defining connections to your target 
+MongoDB database. 
+
+Relational Migrator can use both Atlas and On-Premises URIs. This page 
+provides separate instructions for each deployment type.
+
+Atlas
+-----
+
+Connect to Your Deployment
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To provision user accounts in Atlas you need to be 
+`logged in to the Atlas GUI <https://cloud.mongodb.com/v2/>`__.
+
+Create a New User
+~~~~~~~~~~~~~~~~~
+
+#. In the :guilabel:`Security` section of the left navigation, click :guilabel:`Database Access`.
+#. Click :guilabel:`Add New Database User`.
+#. In the Authentication Method section of the :guilabel:`Add New Database User` modal window, select the box labeled :guilabel:`Password`.
```

Comment:
(markup)
Authentication Method should be a guilabel. 

As part of some future work, it would be helpful to coordinate GUI writer markup practices. This markup is absolutely accurate, but it's a lot of bolded text. It would be interesting to compare processes across writers and teams. This is work for another time, but I thought I'd note that here as well.

Reply:
Yeah agreed, it is alot of bold.